### PR TITLE
Increase granularity of construction codes

### PR DESCRIPTION
**Pull request by bpark1327**
Fixes https://github.com/NREL/OpenStudio-HEScore/issues/208

## Pull Request Description

Increase granularity of construction codes

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [x] Code changes (must work)
- [x] Test exercising your feature or bug fix. Check the coverage report in the build artifacts.
- [x] All other unit tests passing
- [x] ~Update translation docs~

### CODE REVIEW

**Comment by shorowit** [comment]
@bpark1327 Can you create a branch with the same name on OS-HEScore and git subtree pull this in?

**Comment by shorowit** [comment]
... and now that we have a git subtree, move the OS-HEScore/rulesets/HEScoreRuleset/resources lookup files to this repo and update the OS-HEScore ruleset to use the updated paths.

**Review by shorowit — CHANGES_REQUESTED** [review]

**Comment by shorowit** [comment]
@bpark1327 Can you check or ~strikethrough~ the PR checklist boxes?

**Review by shorowit — APPROVED** [review]

**Review by nmerket — COMMENTED** [review]
How are you calculating the assembly R-values? 

Also, this is going to conflict my knee wall stuff. I think this should merge first and I can fix that over there before merging that.

File: hescorehpxml/base.py [hunk 1]
```diff
@@ -180,15 +180,25 @@ def wall_round_to_nearest(*args):
             except RoundOutOfBounds:
                 raise TranslationError('Wall R-value outside HEScore bounds, wall id: %s' % wallid)
 
-        # construction type
-        wall_type = xpath(hpxmlwall, 'name(h:WallType/*)', raise_err=True)
-        assembly_eff_rvalue = None
-
         # Assembly effective R-value or None if element not present
+        assembly_eff_rvalue = None
         assembly_eff_rvalue = self.get_wall_assembly_rvalue(hpxmlwall, hpxmlwall)
```
[the remark this answers is not in the document]
Comment (bpark1327) [reply hescorehpxml/base.py:186]: You're right. Not sure why I didn't remove it. 
```suggestion
        assembly_eff_rvalue = self.get_wall_assembly_rvalue(hpxmlwall, hpxmlwall)

```

**Comment by bpark1327** [comment]
> How are you calculating the assembly R-values?

A linear correlation is expected between the cavity R-value and the assembly R-value because the assembly R-value is the cavity R-value plus a constant R-value such as air film, carpet, etc.
I found a correlation between cavity R-value and assembly R-value using the existing lookup values. Then, I used the correlation to increase the granularity of the lookup values.

**Pull request merged**